documentation/source/user_section/command-line/sct_deepseg.rst

```diff
@@ -99,17 +102,17 @@ Pathology segmentation can be performed by running the following sample command:
 
 .. code::
 
-   sct_deepseg lesion_sci_t2 -i input.nii.gz
+   sct_deepseg lesion_ms -i input.nii.gz
 
-You can replace "``lesion_sci_t2``" with any of the task names in the table below to perform different tasks. Click on a task below for more information.
+You can replace "``lesion_ms``" with any of the task names in the table below to perform different tasks. Click on a task below for more information.
 
 
 .. list-table::
    :align: center
    :widths: 25 25 25
 
-   * - |lesion_sci_t2| ``lesion_sci_t2``
-     - |lesion_ms| ``lesion_ms``
+   * - |lesion_ms| ``lesion_ms``
+     - |lesion_sci_t2| ``lesion_sci_t2``
```

Comment:
This PR should probably not change the order of these models.

Updates to `sct_deepseg` (from the meeting feedback) are being taken care of in PR #4852.

This PR (#4832) should only focus on changes related to the `graymatter` model.

Reply:
Fixed in [d93606c](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4832/commits/d93606cd0a5ae7788fe6c32807a1f437a66db961).